servers/src/main/java/tachyon/worker/block/BlockStore.java

```diff
@@ -176,6 +176,13 @@ TempBlockMeta createBlockMeta(long userId, long blockId, BlockStoreLocation loca
   void accessBlock(long userId, long blockId) throws IOException;
 
   /**
+   * Get the block metadata manager.
+   *
+   * @return block metadata manager
+   */
+  BlockMetadataManager getBlockMetadataManager();
```

Comment:
Instead of exposing the metadata manager, could we just expose the two functions we need? I think its best to keep the metadata manager isolated to just this class.